webpack/plugins.js

```diff
@@ -1,11 +1,8 @@
-'use strict';
-
-const
-    webpack = require('webpack'),
-    env = require('../env'),
-    layout = require('../layout'),
-    Html = require('html-webpack-plugin'),
-    ExtractTextPlugin = require('extract-text-webpack-plugin');
+var webpack = require('webpack');
```

Comment:
vars usage violates our style-guide,
Please use const where possible, and let when mutation is neseccary